driver/src/price_finding/linear_optimization_price_finder.rs

```diff
@@ -370,8 +365,7 @@ pub mod tests {
             "orders": [
                 {
                     "execSellAmount": "0",
-                    "execBuyAmount": "0a",
```

Comment:
What is the `a` for here?

Reply:
It's to make the buy volume not parseable as this test is checking that `serialize_result_fails_if_order_buy_volume_not_parseable `